src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    {(person.sex === 'm' && person.isMarried)
+      ? (
+        <p className="Person__partner">
+          {`${person.partnerName} is my wife`}
+        </p>
+      )
+      : ''}
+
+    {(person.sex === 'f' && person.isMarried)
+      ? (
+        <p className="Person__partner">
+          {`${person.partnerName} is my husband`}
+        </p>
+      )
+      : ''}
+
+    {(!person.isMarried)
+      ? (
+        <p className="Person__partner">
+          I am not married
+        </p>
+      ) : ''}
```

Comment:
you can create a helper something like
```
const partner = sex === 'm'
    ? `${partnerName} is my wife`
    : `${partnerName} is my husband`;
```
    
   and simplify your code buy using this helper
   something like
   `{isMarried ? partner : 'I am not married'}`